.github/workflows/update_remote_docs.yml

```diff
@@ -0,0 +1,19 @@
+name: Trigger build and deploy of Plone 6 documentation
+
+on:
+  push:
+    branches:
+      - "master"
+    paths:
+      - "docs/*"
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          curl -X POST \
+          -H "Authorization: Bearer ${{secrets.DOCUMENTATION_BUILD_APPLICATION_KEY}}" \
```

Comment:
@esteele For future reference, where is this app defined? I thought I might find it in https://github.com/organizations/plone/settings/apps, but I didn't.

Reply:
It's on the Plone Foundation user account, not the organization.